client/package.json

```diff
@@ -36,9 +37,9 @@
     "typeface-roboto": "^0.0.75"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
```

Comment:
Why did you need these script overrides?

Reply:
To extend webpack config.. I didn't want to replace the config provided by create-react-app.
And I needed to configure webpack for aliases